treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -64,6 +66,23 @@ def _add_series(pc, project_name, signature_hash, signature_props, verbosity):
         time_interval=PerformanceTimeInterval.ONE_YEAR)[signature_hash]
 
     with transaction.atomic():
```

Comment:
We don't need a transaction.atomic here, since that was intended for the one-by-one case below.